server/pulp/server/webservices/views/serializers/__init__.py

```diff
@@ -279,7 +280,7 @@ def translate_criteria(self, model, crit):
             crit_dict['sort'] = sort
         if crit.fields:
             crit_dict['fields'] = [self._translate(model, field) for field in crit.fields]
-        return criteria.Criteria.from_dict(crit_dict)
+        return Criteria.from_dict(crit_dict)
```

Comment:
Since you understand the problem better than I do, do you mind filing the issue?